spec/controllers/competitions_controller_spec.rb

```diff
@@ -885,10 +885,10 @@
         expect(competition.results_posted_by).to eq wrt_member.id
       end
 
-      it "sends notifications of id claim possibility to newcomers" do
+      it "sends notifications of id claim possibility to first-timers" do
         competition = FactoryBot.create(:competition, :registration_open)
-        FactoryBot.create_list(:registration, 2, :accepted, :newcomer, competition: competition)
-        FactoryBot.create_list(:registration, 3, :pending, :newcomer, competition: competition)
+        FactoryBot.create_list(:registration, 2, :accepted, :first_timer, competition: competition)
+        FactoryBot.create_list(:registration, 3, :pending, :first_timer, competition: competition)
```

Comment:
This is now a bit inconsistent with the `newcomer` / `newcomer_month_eligible` thingy above

Reply:
Yep this is the result of an over-eager grep/replace, I think